contracts/RibbonStakingRewards.sol

```diff
@@ -27,7 +27,7 @@ contract StakingRewards is
     IERC20 public stakingToken;
     uint256 public periodFinish = 0;
     uint256 public rewardRate = 0;
-    uint256 public rewardsDuration = 7 days;
+    uint256 public rewardsDuration = 30 days;
     uint256 public lastUpdateTime;
     uint256 public rewardPerTokenStored;
 
```

Comment:
also, you will notice that in notifyRewardAmount, periodFinish is [defined](https://github.com/ribbon-finance/token/blob/dbe7357bca37d985647b9574b1548aae1bea362f/contracts/RibbonStakingRewards.sol#L206) based on current block timestamp. This is fine as long as we set notifyRewardAmount close to startEmissionAnyway. 

However, we can set it to periodFinish = startEmission.add(rewardDuration) for precision. but this means we cannot call notifyRewardAmount later on, so we need to be sure we will not be extending the LM program (extending entails calling notifyRewardAmount with new reward). this has been largely irrelevant until discussion of potential extension of program came up.